modules/onscreengui/src/renderproperties.cpp

```diff
@@ -46,13 +46,22 @@ void renderTooltip(Property* prop) {
     }
 }
 
-void executeScript(const std::string& id, const std::string& value) {
+void executeScriptSingle(const std::string& id, const std::string& value) {
```

Comment:
I understand that this is more efficient since it does not use regex, but is it ever used?